### ISSUE TITLE
ReferenceError: Invalid left-hand side in

### ISSUE THREAD

**Issue by pashminakazi** [apache/royale-asjs#651]
Project Details are :
https://paste.apache.org/cpo8y
Test case is available on VDI.
Go to General Ledger from Select in Menu
Go to GL Analysis -> Budget Type from Setups in Menu
File : com/dbz/modules/GL/dbzGLAnalysis1Setup.mxml

In dbzGLAnalysis1Setup we are using dbzSetupList.mxml which is in Lib

First you have to save a record and then resave it with some changes in description field.description field will not be updated and this exception occurs.
![GLanalysis issue](https://user-images.githubusercontent.com/42200979/71709981-db82d280-2e1b-11ea-9d8e-830a28cc4938.gif)

**Comment by aharui** [comment]
Are you sure this module and its copy of dbzSetupList were re-compiled and re-deployed?  It looks like it is the same as #636

**Comment by pashminakazi** [comment]
Yes these are recompiled and re-deployed.

**Comment by aharui** [comment]
On the VDI, I verified that the compiler is generating the correct code.  It appears that your are not using the latest compiler.  You must:
1) get the latest compiler sources
2) run ant in royale-compiler
3) run ant in royale-asjs
4) run ant in DBIZCOMLIB/src
5) build and deploy all modules that use dbzSetupList

The VDI was not correctly configured.  I think step 3 was skipped.  I ran all steps in order and verified that the output dbzSetupList.js is now correct.
